keylime/ima.py

```diff
@@ -204,9 +205,10 @@ def _validate_ima_buf(exclude_regex, allowlist, ima_keyrings: ima_file_signature
     return failure
 
 
-def _process_measurement_list(agentAttestState, lines, lists=None, m2w=None, pcrval=None, ima_keyrings=None, boot_aggregates=None):
+def _process_measurement_list(agentAttestState, lines, lists=None, m2w=None, pcrval=None, ima_keyrings=None,
+                              boot_aggregates=None, hash_alg=Hash.SHA1):
```

Comment:
Nit: Should we make this parameter mandatory, meaning no fallback value, and put it after `lists` ?

Reply:
Yeah fixed.